docs/OV_Runtime_UG/integrate_with_your_application.md

```diff
@@ -240,24 +233,21 @@ The example uses CMake for project configuration.
        ...      
    ```
 
-2. **Include OpenVINO™ Runtime libraries** in `project/CMakeLists.txt`
+### Include OpenVINO™ Runtime libraries in `project/CMakeLists.txt`
 
    @snippet snippets/CMakeLists.txt cmake:integration_example
 
 To build your project using CMake with the default build tools currently available on your machine, execute the following commands:
 
-> **NOTE**: Make sure you set environment variables first by running `<INSTALL_DIR>/setupvars.sh` (or `setupvars.bat` for Windows). Otherwise the `OpenVINO_DIR` variable won't be configured properly to pass `find_package` calls.
-
 ```sh
 cd build/
 cmake ../project
 cmake --build .
 ```
-You can also specify additional build options (e.g. to build CMake project on Windows with a specific build tools). Please refer to the [CMake page](https://cmake.org/cmake/help/latest/manual/cmake.1.html#manual:cmake(1)) for details.
 
-## Run Your Application
+> **NOTE**: Make sure you set environment variables first by running `<INSTALL_DIR>/setupvars.sh` (or `setupvars.bat` for Windows). Otherwise the `OpenVINO_DIR` variable won't be configured properly to pass `find_package` calls.
 
-Congratulations, you have made your first application with OpenVINO™ toolkit, now you may run it.
+You can also specify additional build options (e.g. to build CMake project on Windows with a specific build tools). Please refer to the [CMake page](https://cmake.org/cmake/help/latest/manual/cmake.1.html#manual:cmake(1)) for details.
```

Comment:
I don't feel strongly about this, but could this sentence be removed? Surely people who use CMake for a project understand they can use CMake options. Maybe link to the CMake docs in line 223 and then delete this line?